src/Hooks/LandingPages.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+use WPML\FP\Obj;
+
+class LandingPages implements \IWPML_Frontend_Action, \IWPML_Backend_Action, \IWPML_DIC_Action {
+
+	const POST_TYPE = 'e-landing-page';
+
+	/** @var \SitePress $sitepress */
+	private $sitepress;
+
+	public function __construct( \SitePress $sitepress ) {
+		$this->sitepress = $sitepress;
+	}
+
+	public function add_hooks() {
+		add_filter( 'post_type_link', [ $this, 'adjustLink' ], PHP_INT_MAX, 2 );
+	}
+
+	/**
+	 * @see \Elementor\Modules\LandingPages\Module::remove_post_type_slug
+	 *
+	 * @param string   $postUrl
+	 * @param \WP_Post $post
+	 *
+	 * @return string
+	 */
+	public function adjustLink( $postUrl, $post ) {
+		if ( self::POST_TYPE === $post->post_type ) {
+			$homeUrl          = get_home_url();
+			$urlParts         = wp_parse_url( $homeUrl );
+			$urlParts['path'] = trailingslashit( Obj::prop( 'path', $urlParts ) ) . $post->post_name . '/';
```

Comment:
@strategio what happens if the permalink structure doesn't have a trailing slash?

Reply:
>@strategio what happens if the permalink structure doesn't have a trailing slash?

`trailingslashit` will always add a trailing slash @dgwatkins (maybe you are mixing with `usertrailingslashit`), see below:

```php
function trailingslashit( $string ) {
	return untrailingslashit( $string ) . '/';
}
```

>or if its default permalink structure ...

I didn't test this... what I did is just transposing what Elementor is doing:

```php
return get_home_url() . '/' . $post->post_name . '/';
```

But I will test again (maybe Elementor does not allow the default permalink structure).

Reply:
Good catch @dgwatkins, indeed Elementor's link filter is not loaded with default permalinks (I added a fix commit 
7dda8b7).

Then, I overlooked Elementor filter and noticed it filters only "published" landing pages, so I added one extra commit for this 82308c3.